packages/schematics/angular/workspace/files/package.json.template

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@angular/cli": "<%= '~' + version %>",
     "@angular/compiler-cli": "<%= latestVersions.Angular %>",<% if (!minimal) { %>
-    "@types/jasmine": "~3.6.0",<% } %>
+    "@types/jasmine": "~3.8.0",<% } %>
     "@types/node": "^12.11.1",<% if (!minimal) { %>
     "jasmine-core": "~3.7.0",
```

Comment:
Should `jasmine-core` be updated as well?

Reply:
Yes